app/controllers/api/v1/base_controller.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class BaseController < ApplicationController
+      include Api::OpenidConnect::ProtectedResourceEndpoint
+
+      protected
+
+      rescue_from Exception do |e|
+        logger.error e.message
+        logger.error e.backtrace.join("\n")
+        render json: error_body(500, e.message), status: 500
```

Comment:
Rails/HttpStatus: Prefer `:internal_server_error` over `500` to define HTTP status code.